src/main/java/org/zalando/nakadi/validation/schema/CompatibilityModeChangeConstraint.java

```diff
@@ -14,7 +14,7 @@
     final Map<CompatibilityMode, List<CompatibilityMode>> allowedChanges = ImmutableMap.of(
             CompatibilityMode.COMPATIBLE, Lists.newArrayList(CompatibilityMode.COMPATIBLE),
             CompatibilityMode.FORWARD, Lists.newArrayList(CompatibilityMode.FORWARD, CompatibilityMode.COMPATIBLE),
-            CompatibilityMode.NONE, Lists.newArrayList(CompatibilityMode.NONE)
+            CompatibilityMode.NONE, Lists.newArrayList(CompatibilityMode.NONE, CompatibilityMode.FORWARD)
```

Comment:
Ok, wanted to make sure it's a conscious decision.

The background for my question was: For us schema evolution is rather slow as we automatically register/update event types on service deployments. In this mode requiring an extra evolution means an extra release.